content/blog/tidyclust-0-1-0/index.Rmd

```diff
@@ -0,0 +1,150 @@
+```
+
+This blog post will introduce tidyclust, how to use it with the rest of tidymodels, and how we can interact and evaluate the fitted cluster models.
+
+```{r setup}
+library(tidymodels)
+library(tidyclust)
+```
+
+## Specifying cluster models
+
+The first thing we need to do is decide on the type of clustering model we want to fit. The pkgdown site provides a [list of all clustering specifications](https://tidyclust.tidymodels.org/reference/index.html#specifications) provided by tidyclust. We are slowly adding more types of models and [suggestions in issues](https://github.com/tidymodels/tidyclust/issues) are highly welcome!
+
+We will use a K-Means model for these examples using `k_means()` to create a specification. tidyclust as with the rest of tidymodels tries to use informative names for functions and arguments, so we will be using `num_clusters` instead of `k`
+
+```{r}
+kmeans_spec <- k_means(num_clusters = 4) %>%
+  set_engine("ClusterR")
+kmeans_spec
+```
+
+We can use the `set_engine()`, `set_mode()`, and `set_args()` we are familiar with from parsnip. The specification itself isn't worth much if we don't apply it to some data. We will use the ames data set from the modeldata package
+
+```{r}
+data("ames", package = "modeldata")
+```
+
+this data set contains a number of categorical variables that unaltered can't be used with a k-means model. Some light preprocessing can be done using the recipes package.
+
+```{r}
+rec_spec <- recipe(~ ., data = ames) %>%
+  step_dummy(all_nominal_predictors()) %>%
+  step_zv(all_predictors()) %>%
+  step_normalize(all_numeric_predictors()) %>%
+  step_pca(all_numeric_predictors(), threshold = 0.8)
+```
+
+This recipe ensures that all the variables are numeric, normalized and have PCA applied to them to remove some of the correlation and number of features. Notice how we didn't specify an outcome as clustering models are unsupervised, meaning that we don't have outcomes.
+
+These two specifications can be combined in a `workflow()` 
+
+```{r}
+kmeans_wf <- workflow(rec_spec, kmeans_spec)
+```
+
+And fit them together
+
+```{r}
+kmeans_fit <- fit(kmeans_wf, data = ames)
+kmeans_fit
+```
+
+We have arbitrarily set the number of clusters to 4 above. But if we wanted to figure out what values would be "optimal" we would have to fit multiple models. We can do this with `tune_cluster()`, for this we need to use `tune()` to specify that `num_clusters` is the argument we want to try with multiple values
+
+```{r}
+kmeans_spec <- kmeans_spec %>% 
+  set_args(num_clusters = tune())
+
+kmeans_wf <- workflow(rec_spec, kmeans_spec)
+kmeans_wf
+```
+
+Using bootstraps to fit multiple models for each value of `num_clusters` and we can use `tune_cluster` much the same way we use `tune_grid()`
+
+```{r}
+set.seed(1234)
+boots <- bootstraps(ames, times = 10)
+
+tune_res <- tune_cluster(
+  kmeans_wf,
+  resamples = boots
+)
+```
+
+and the respective `collect_*()` functions work as with tune output
```

Comment:
You capitalize the analogous "and" in a previous line. I would probably transition this to a full sentence. :)